sceptre/stack.py

```diff
@@ -612,6 +632,21 @@ def _get_role_arn(self):
         else:
             return {}
 
+    def _get_stack_timeout(self):
```

Comment:
I find this name a bit misleading since it appears to be used elsewhere s a way to wrap the stack timeout for use as a keyword argument.

Reply:
I also follow the "convention" here because of the following method:
```python
def _get_role_arn(self):
```
which does exactly the same thing.

Reply:
In that case, it's fine as it is (I still think the test case is unnecessary, though).